site/content/docs/5.2/getting-started/vite.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: docs
+title: "Bootstrap & Vite"
+description: The official guide for how to include and bundle Bootstrap's CSS and JavaScript in your project using Vite.
+group: getting-started
+toc: true
+---
+
+{{< callout >}}
+**Want to skip to the end?** Download the source code and working demo for this guide from the [twbs/examples repository](https://github.com/twbs/examples/tree/main/vite). You can also [open the example in StackBlitz](https://stackblitz.com/github/twbs/examples/tree/main/vite?file=index.html) for live editing.
+{{< /callout >}}
+
+## Setup
+
+We're building a Vite project with Bootstrap from scratch, so there are some prerequisites and up front steps before we can really get started. This guide requires you to have Node.js installed and some familiarity with the terminal.
+
+<br>
+
+1. **Create a project folder and setup npm.** We'll create the `my-project` folder and initialize npm with the `-y` argument to avoid it asking us all the interactive questions.
+
+   ```sh
+   mkdir my-project && cd my-project
+   npm init -y
+   ```
+
+2. **Install Vite.** Next we need to install our Vite development dependencies: `vite` for the core of Vite. We use `--save-dev` to signal that these dependencies are only for development use and not for production.
+
+   ```sh
+   npm i --save-dev vite
+   ```
+
+3. **Install Bootstrap.** Now we can install Bootstrap. We'll also install Popper since our dropdowns, popovers, and tooltips depend on it for their positioning. If you don't plan on using those components, you can omit Popper here.
+
+   ```sh
+   npm i --save bootstrap @popperjs/core
+   ```
+4. **Install additional dependencies.** In addition to Vite and Bootstrap, we need a few more dependencies to properly import and bundle Bootstrap's CSS and JS with Vite. These include Sass<!--, some loaders, and Autoprefixer-->.
+
+<!--npm i save-dev autoprefixer css-loader postcss-loader sass sass-loader style-loader-->
+
+   ```sh
+   npm i --save-dev sass
+   ```
+
+Now that we have all the necessary dependencies installed and setup, we can get to work creating the project files and importing Bootstrap.
+
+## Project structure
+
+We've already created the `my-project` folder and initialized npm. Now we'll also create our `src` and `dist` folders to round out the project structure. Run the following from `my-project`, or manually create the folder and file structure shown below.
+
+```sh
+mkdir {dist,src,src/js,src/scss}
+touch dist/index.html src/js/main.js src/scss/styles.scss src/scss/_custom.scss vite.config.js
```

Comment:
Even in Webpack guide, we never use `_custom.scss` file. So we can maybe remove it from all guides.